src/Home/Home.js

```diff
@@ -47,10 +47,12 @@ class Home extends React.Component {
     filter: {},
   }
 
-  canShow = (project) => {
+  canShow = project => {
+    // eslint-disable-next-line guard-for-in, no-restricted-syntax
```

Comment:
I left the following lines untouched and added `// eslint-disable-next-line` to suppress the following errors:
```
   51:5   error  The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype                                                             guard-for-in
   51:5   error  for..in loops iterate over the entire prototype chain, which is virtually never what you want. Use Object.{keys,values,entries}, and iterate over the resulting array  no-restricted-syntax
   54:9   error  Unexpected use of continue statement                                                                                                                                   no-continue
```

Do you think this should be rewritten without `for`-`in`?